src/src/common.cpp

```diff
@@ -11,14 +11,16 @@ expresslrs_mod_settings_s ExpressLRS_AirRateConfig[RATE_MAX] = {
     {1, RADIO_TYPE_SX127x_LORA, RATE_LORA_100HZ_8CH, SX127x_BW_500_00_KHZ, SX127x_SF_6, SX127x_CR_4_8, TLM_RATIO_1_32, 4, 10000,  8, OTA8_PACKET_SIZE, 1},
     {2, RADIO_TYPE_SX127x_LORA, RATE_LORA_100HZ,     SX127x_BW_500_00_KHZ, SX127x_SF_7, SX127x_CR_4_7, TLM_RATIO_1_32, 4, 10000,  8, OTA4_PACKET_SIZE, 1},
     {3, RADIO_TYPE_SX127x_LORA, RATE_LORA_50HZ,      SX127x_BW_500_00_KHZ, SX127x_SF_8, SX127x_CR_4_7, TLM_RATIO_1_16, 4, 20000, 10, OTA4_PACKET_SIZE, 1},
-    {4, RADIO_TYPE_SX127x_LORA, RATE_LORA_25HZ,      SX127x_BW_500_00_KHZ, SX127x_SF_9, SX127x_CR_4_7, TLM_RATIO_1_8,  2, 40000, 10, OTA4_PACKET_SIZE, 1}};
+    {4, RADIO_TYPE_SX127x_LORA, RATE_LORA_25HZ,      SX127x_BW_500_00_KHZ, SX127x_SF_9, SX127x_CR_4_7, TLM_RATIO_1_8,  2, 40000, 10, OTA4_PACKET_SIZE, 1},
+    {5, RADIO_TYPE_SX127x_LORA, RATE_DVDA_50HZ,      SX127x_BW_500_00_KHZ, SX127x_SF_6, SX127x_CR_4_7, TLM_RATIO_1_64, 2,  5000,  8, OTA4_PACKET_SIZE, 4}};
 
 expresslrs_rf_pref_params_s ExpressLRS_AirRateRFperf[RATE_MAX] = {
     {0, RATE_LORA_200HZ,     -112,  4380, 3000, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(3.0)},
     {1, RATE_LORA_100HZ_8CH, -112,  6690, 3500, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(3.0)},
     {2, RATE_LORA_100HZ,     -117,  8770, 3500, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(2.5)},
     {3, RATE_LORA_50HZ,      -120, 18560, 4000, 2500, 600, 5000, SNR_SCALE(-1), SNR_SCALE(1.5)},
-    {4, RATE_LORA_25HZ,      -123, 29950, 6000, 4000,   0, 5000, SNR_SCALE(-3), SNR_SCALE(0.5)}};
+    {4, RATE_LORA_25HZ,      -123, 29950, 6000, 4000,   0, 5000, SNR_SCALE(-3), SNR_SCALE(0.5)},
+    {0, RATE_DVDA_50HZ,      -112,  4380, 3000, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(3.0)}};
```

Comment:
I know the index isn't used but if we are having it in the structure then it should at least be correct. So it should be 5 at the start 😄

Reply:
Suppose 😄